modules/telestion-api/src/main/java/org/telestion/api/message/JsonMessage.java

```diff
@@ -39,7 +41,8 @@
 				handler.handle((clazz.cast(jsonObject.mapTo(msgClazz))));
 				return true;
 			} catch (ClassNotFoundException e) {
-				e.printStackTrace();
+				logger.error("Error while converting JSON into JsonMessage. The requested message is not available on the "
```

Comment:
Please reduce this to warn. Then it is fine :)

Reply:
But isn't this a critical error?

Reply:
No, it is not. See my notes above. It is just a hint that somebody is dropping some unknown content at you. No need to worry too much.